features/step_definitions/clear_data_on_signout_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^I am on the Money Manager tool$/) do
+  money_manager_page.load
+end
+
+Then(/^I complete the series of questions$/) do
+  money_manager_page.received_first_payment_true.click
+  money_manager_page.single_or_in_couple_single.click
+  money_manager_page.submit.click
+end
+
+Given(/^I get my results$/) do
+  expect(page.current_path).to eql('/en/tools/money-manager/to-read')
```

Comment:
In this story we specifically need to check that we see the 'to-read' page which is different from `money_manager_page`'s url. Is there a way we can do that with the same object? Or should we create a `managing_money_page` object for that?